CHANGELOG.md

```diff
@@ -49,15 +50,26 @@ and removing some features that no longer seem necessary.
   browser normalization, is no longer included. We now recommend
   using [CSS Remedy](https://github.com/jensimmons/cssremedy).
 - BREAKING: The old "core" module has been broken into individual modules
-  for "utilities" (extending several Sass modules),
-  "tokens" (the map syntax parser),
-  "variables" (converting Sass variables & maps to CSS custom properties),
-  and "ratios" (for typographic scales and layout aspect ratios)
+  for [utilities][utils] (extending several Sass modules),
+  [tokens][token] (the map syntax parser),
+  [variables][vars] (converting Sass variables & maps to CSS custom properties),
+  and [ratios][ratio] (for typographic scales and layout aspect ratios)
```

Comment:
All of these links are currently broken, though I think they will work once this is fully published since they are the new pages.

Reply:
yes, these should work once this is fully published. That will be something good to test right after we merge.